kolena/_utils/krequests.py

```diff
@@ -52,6 +52,9 @@
 # Using the Retry object to configure a backoff which is not supported by using an int here.
 MAX_RETRIES = Retry(total=3, connect=3, read=0, redirect=0, status=0, backoff_factor=2)
 
+# This retries for read errors in addition to the ones mentioned above.
+MAX_RETRIES_WITH_READ = Retry(total=3, connect=3, read=3, redirect=0, status=0, backoff_factor=2)
```

Comment:
nit: Might call this `MAX_IDEMPOTENT_RETRIES`? Maybe just lack of clarity on my part of what exactly constitutes a `ReadTimeoutError`